korge/src/common/korlibs/korge/view/SolidTriangle.kt

```diff
@@ -0,0 +1,48 @@
+package korlibs.korge.view
+
+import korlibs.image.color.*
+import korlibs.math.geom.*
+import korlibs.memory.*
+
+
+class SolidTriangle(
+  p1: Point, p2: Point, p3: Point,
+  color: RGBA = Colors.WHITE,
+) : Mesh(
+  vertices = Float32Buffer(
+    floatArrayOf(
+      p1.x.toFloat(), p1.y.toFloat(),
+      p2.x.toFloat(), p2.y.toFloat(),
+      p3.x.toFloat(), p3.y.toFloat(),
+    )
+  ),
+  indices = Uint16Buffer(UShortArrayInt(shortArrayOf(0, 1, 2))),
+  uvs = Float32Buffer(12)
+) {
+  var p1: Point = p1
+    set(value) {
+      updatedVertices()
```

Comment:
This doesn't seem to work - updating the p1 is not updating the rendered triangle